src/helpers/utils.js

```diff
@@ -5,33 +5,45 @@ const { repoMapping } = require("../constants");
  * if `except` is provided, it should be a string of characters to not escape
  * https://core.telegram.org/bots/api#markdownv2-style
  */
-const escapeMarkdown = (str, except = "") => {
+const escapeMarkdown = (str, except = "") =>
+{
   const all = "_*[]()~`>#+-=|{}.!\\".split("").filter((c) => !except.includes(c));
   const regExSpecial = "^$*+?.()|{}[]\\";
   const regEx = new RegExp("[" + all.map((c) => (regExSpecial.includes(c) ? "\\" + c : c)).join("") + "]", "gim");
   return str.replace(regEx, "\\$&");
 };
 
-const extractNumberWithoutPrefix = (text) => {
+const extractNumberWithoutPrefix = (text) =>
+{
   const numberWithoutPrefix = text.replace(/^(-)?\d{3}/, "");
   return numberWithoutPrefix.length === 10 ? numberWithoutPrefix : null;
 };
 
-const cleanMessage = (text) => {
+const cleanMessage = (text) =>
+{
   // Remove all occurrences of @tag
   const cleanedText = text.replace(/@\w+/g, "");
 
   // Remove all occurrences of links (http and https)
   return cleanedText.replace(/(https?:\/\/[^\s]+)/g, "");
```

Comment:
Why are links removed?

Reply:
> Why are links removed?

It takes texts from links and the result was not great, so at some point we clean the message and then add back what was cleaned to give great results

Reply:
I don't understand. It extracts the link href and then places it directly in the GitHub issue body?